src/main/java/org/jenkinsci/plugins/periodicbackup/ConfigOnly.java

```diff
@@ -112,12 +112,7 @@ private void addUserFiles(File rootDir, List<File> filesToBackup) throws Periodi
         File usersDir = new File(rootDir, "users");
         if (usersDir.exists() && usersDir.isDirectory()) {
             // Each user directory should have a config.xml file
-            File[] dirsInUsers = usersDir.listFiles((FileFilter) FileFilterUtils.directoryFileFilter());
-            if (dirsInUsers == null) {
-                // First one may happen only due to the race condition
-                throw new PeriodicBackupException("Job directory path does not denote or there is an I/O error");
-            }
-            
+            File[] dirsInUsers =  Util.listFiles(usersDir, FileFilterUtils.directoryFileFilter());
```

Comment:
Hm? Just replaced it by the `Util` method which I've created. A leftover from the original fix attempts